PlaygroundService/Grains/IPlaygroundGrain.cs

```diff
@@ -6,4 +6,6 @@ namespace PlaygroundService.Grains;
 public interface IPlaygroundGrain : IGrainWithStringKey
 {
     public Task<(bool, string)> BuildProject(string directory);
+    public Task<bool> DelData(string zipFile, string extractPath);
+    public Task<string> GenerateZip(string template, string templateName);
```

Comment:
Rename to `GenerateTemplate`. This might be more intuitive.

Reply:
done